src/vehicle.cpp

```diff
@@ -1727,7 +1727,7 @@ bool vehicle::can_mount(int const dx, int const dy, const vpart_id &id) const
 
     // curtains must be installed on (reinforced)windshields
     // TODO: do this automatically using "location":"on_mountpoint"
-    if (part.has_flag("CURTAIN")) {
+    if (part.has_flag("WINDOW_CURTAIN")) {
```

Comment:
This file looks like it hasn't been astyled yet, but in the future, you'll want this line to look like this:
```cplusplus
    if( part.has_flag( "DOOR_MOTOR" ) ) {
```

or you'll end up with build failures.

If you want to try astyling it now (it shouldn't fail as this file seems to be half-astyled and half not), it should look like:

```cplusplus
    if( part.has_flag( "WINDOW_CURTAIN" ) ) {
```